csfieldguide/chapters/content/en/big-data/sections/the-three-vs.md

```diff
@@ -0,0 +1,53 @@
+# Big Data and The Three V's
+
+Being able to find patterns in huge amounts of data can be incredibly valuable for making decisions, for example for setting prices, making recommendations, and detecting suspicious activities.
+This is why Big Data is a bit of a buzzword in tech and in commercial industries these days.
+But how big does it have to be to be called "*Big Data*"?
+
+Big Data is a phrase used to mean an amount of data that is so large and incoming so quickly that it is difficult to store and analyse using traditional sequential programming and storage techniques.
+Big Data has three main qualities: Volume, Variety, and Velocity (often referred to as the "3Vs").
+
+## Volume
+
+Volume refers to the quantity of data, and when it comes to Big Data there is a lot of it. Have a look at this website showing what happens on the internet [every second](https://everysecond.io/the-internet) to get an idea of just how much data is constantly produced online!
+
+Your phone might have about 64GB of storage space and it takes a long time for the typical smartphone user to run out of space.
+A laptop might have 1TB of storage, and some users will never get close to using all that space.
+It wasn't long ago that running out of space was a common problem for all users, but advances in engineering of transistor density have continued at a steady pace.
+The capacity of a single computer's integrated circuit doubles about every two years ([Moore's Law](https://en.wikipedia.org/wiki/Moore%27s_law)), so if you're reading this several years after it was written 1TB might sound quite small!
```

Comment:
This semi-contradicts the same factoid from a curiosity in [the previous chapter](https://www.csfieldguide.org.nz/en/chapters/software-engineering/whats-the-big-picture/). Perhaps align the two a bit more

Reply:
I'm not quite sure what part of these is contradictory?

This part has changed a little based on feedback from Tim, it's now:

> It wasn't long ago that running out of space was a common problem for all users, but advances in engineering of transistor density have continued at a steady pace.
Since the 60's the capacity of a single computer's integrated circuit has been doubling roughly every two years ([Moore's Law](https://en.wikipedia.org/wiki/Moore%27s_law)), so if this continues and you're reading this several years after it was written, 1TB might sound quite small!

Reply:
This is actually fine, I was misinterpreting something